stylesheet.css

```
@@ -0,0 +1,34 @@
+body{
+    /*text-align: center;*/
+    color:white;
+        background-image: linear-gradient(
+        115deg,
+        rgba(58, 58, 158, 0.8),
+        rgba(136, 136, 206, 0.7)
+      ),
+      url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg);
+}
+form{
```

I suggest making sure there is a space between classes and curly brackets. 
```suggestion
form {
```

---

As I told you before, kindly make sure you explain well why you requested the changes and please, be more polite in your comments. 😄

---

Good job with your styling! Also, remember that to make your code easier to read, you need to leave spaces between semicolons and styling, as follows:
```suggestion
form {
```